flags/BUILD.bazel

```diff
@@ -10,7 +10,7 @@ load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
 # Define "--//flags:ovs" command-line flag
 bool_flag(
     name = "ovs",
-    build_setting_default = True,
+    build_setting_default = False,
```

Comment:
Are we disabling the ovs build to off in bazel with this change?

Reply:
1. We're enabling/disabling some conditional code in krnlmon that's associated with ovs-p4rt. (No other part of the code is affected.)
2. We're only doing this in the standalone Bazel build (when krnlmon is built by itself, within the krnlmon submodule).
3. We're not so much disabling it as changing the default value of a command-line flag. The developer can always override it by specifying `--//flags:ovs=True` on the Bazel command line.
4. The standalone cmake build defaults to disabled. This makes the standalone build have the same defaults as the standalone cmake build.
5. This all started when one of the krnlmon unit tests failed under Bazel but not under CMake ... until I added `-DWITH_OVSP4RT=OFF` to the cmake command line, and it failed there, too. It's a defect in the unit test. Changing the Bazel default to match the standalone CMake default puts the two on equal footing, and allows me to delay investigating the unit test failure while I work on the Bazel build.